### PR TITLE
Migrated CM No Content type view

### PR DESCRIPTION
**Pull request by soupette**
Screenshot 2021-09-09 at 08 11 49

## How to test

1. In packages/core/admin/admin/src/content-manager/pages/App comment the following lines:

```
  if (authorisedModels.length === 0 && models.length > 0 && pathname !== '/content-manager/403') {
    return <Redirect to="/content-manager/403" />;
  }

  // Redirect the user to the create content type page
  if (models.length === 0 && pathname !== '/content-manager/no-content-types') {
    return <Redirect to="/content-manager/no-content-types" />;
  }

  if (!contentTypeMatch && authorisedModels.length > 0) {
    return (
      <Redirect
        to={`${authorisedModels[0].to}${
          authorisedModels[0].search ? `?${authorisedModels[0].search}` : ''
        }`}
      />
    );
  }
```

2. Go to http://localhost:4000/admin/content-manager/no-content-types

### CODE REVIEW

File: packages/core/admin/admin/src/content-manager/pages/App/index.js [hunk 1]
```diff
@@ -20,9 +23,22 @@ const App = () => {
   const { status, collectionTypeLinks, singleTypeLinks, models, refetchData } = useModels();
   const authorisedModels = sortBy([...collectionTypeLinks, ...singleTypeLinks], 'title');
   const { pathname } = useLocation();
+  const { formatMessage } = useIntl();
 
+  // Check with @mfrachet if ok with a11y
   if (status === 'loading') {
-    return <LoadingIndicatorPage />;
+    return (
+      <Main labelledBy="title">
```
Comment (mfrachet) [inline packages/core/admin/admin/src/content-manager/pages/App/index.js:31]: you'll need to put a `aria-busy` on this component

File: packages/core/admin/admin/src/content-manager/pages/App/useModels.js [hunk 2]
```diff
@@ -36,6 +40,13 @@ const useModels = () => {
         )
       );
 
+      notifyStatus(
```
Comment (mfrachet) [inline packages/core/admin/admin/src/content-manager/pages/App/useModels.js:43]: ❤️

File: packages/core/admin/admin/src/content-manager/pages/NoContentType/index.js [hunk 3]
```diff
@@ -1,68 +1,51 @@
...
+          defaultMessage: 'Content',
+        })}
+      />
+      <ContentLayout>
+        <EmptyStateLayout
+          action={
+            <LinkButton
+              variant="secondary"
+              startIcon={<AddIcon />}
+              to="/plugins/content-type-builder/content-types/plugin::users-permissions.user?modalType=contentType&kind=collectionType&actionType=create&settingType=base&forTarget=contentType&headerId=content-type-builder.modalForm.contentType.header-create&header_icon_isCustom_1=false&header_icon_name_1=contentType&header_label_1=null"
```
Comment (mfrachet) [inline packages/core/admin/admin/src/content-manager/pages/NoContentType/index.js:30]: is this link necessary?

**Review by mfrachet — COMMENTED** [review]
Some small comments, when they're taken, you can merge 👍🏻 . Nice job and thank you 😊

File: packages/core/admin/admin/src/content-manager/pages/NoContentType/index.js (hunk 3 above)
Comment (soupette) [reply packages/core/admin/admin/src/content-manager/pages/NoContentType/index.js:30]: Yes ! it's in the new designs...

**Pull request merged**